documentation/modules/overview/con-configuration-points-broker.adoc

```diff
@@ -0,0 +1,33 @@
+// This module is included in:
+//
+// overview/assembly-configuration-points.adoc
+
+// UserStory: Cluster broker configuration
+
+[id="configuration-points-broker_{context}"]
+= Kafka cluster configuration
+
+A Kafka cluster configuration describes how data is stored and accessed.
```

Comment:
I think this might more sense seeing the whole chapter. We talk about common configuration earlier and now we talk about key configuration points for each component. Here, it's Kafka. So I'm highlighting storage and listeners for storage and access. 

I've changed the intro a bit:

A kafka cluster comprises one or more brokers.
For producers and consumers to be able to access topics within the brokers, Kafka configuration must define how data is stored in the cluster and the type of access.

Reply:
Would _how data is stored in the cluster and how data is accessed_ sound better than _how data is stored in the cluster and the type of access_?